independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Webex Integration",
+    "description": "WebEx API for Power Platform\n\n- Setup meeting from Power Platform",
+    "contact": {
+      "name": "UCL",
+      "email": "oscar.hui.21@ucl.ac.uk"
+    }
+  },
+  "host": "webexapis.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/meetings": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@ocarhui is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all responses in your connector swagger file.